base/show.jl

```diff
@@ -514,9 +514,10 @@ function _show_default(io::IO, @nospecialize(x))
 end
 
 function active_module()
-    REPL = REPL_MODULE_REF[]
-    REPL === Base && return Main
-    return invokelatest(REPL.active_module)::Module
+    if !@isdefined(active_repl)
+        return Main
+    end
+    return invokelatest(active_module, active_repl)::Module
```

Comment:
I think you mean to just return it here, not call it

Reply:
In particular, anything that tries to use invokelatest will never work for this

Reply:
Do you mean there needs to be a case to handle `in_pure_context`?